### ISSUE TITLE
Empty rules page

### ISSUE THREAD

**Issue by Casper-Guo** [LakeBlair/RoyalGame#2]
https://github.com/LakeBlair/RoyalGame/blob/4766db9e2a854c250fcda8877ff3f201b1467701/frontend/src/HomePage.js#L20 leads to an empty page.

Suggest redirecting to wikipedia instead.

**Comment by LakeBlair** [comment]
Yes we haven't add link for that part yet. You can make a PR if you'd like.

**Comment by Casper-Guo** [comment]
![where_rules](https://github.com/LakeBlair/RoyalGame/assets/89810860/6e232aad-fb26-4b72-bc75-6d8202a37313)
